packages/@ore-ui/gameface-image-preloader/README.md

```diff
@@ -0,0 +1,30 @@
+# @ore-ui/gameface-image-preloader
+There isn't support for the onLoad event in Gameface right now. Therefore, it's hard to create a smooth UI where
```

Comment:
Formatting: we probably want a line break here. (unfortunately we don't have a prettier check yet for Readme files).

Reply:
I had a look at the other README files, and it didn't look like we followed any convention for line length. The root README has a line with more than 300 characters. Therefore, I originally set the text width to 120 (same as the code). I have now set it to 80 and reformatted this file.